Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  URBNSwiftAccordionTableViewController.swift
+//  URBNDataSource
+//
+//  Created by Dustin Bergman on 12/15/15.
+//  Copyright © 2015 Joe. All rights reserved.
+//
+
+import UIKit
+
+typealias CellTappedBlock = () -> Void
+
+class URBNSwiftAccordionTableViewController: UITableViewController {
+    
+    var adapter: URBNAccordionDataSourceAdapter?
+    @IBOutlet var stepper: UIStepper?
```

Comment:
More of a personal preference.  I like to force unwrap I outlets when I know that I expect them to be wired.  Saves some "?" Later on.   


Not sure what the team thinks